docs/Flux_Functions.md

```diff
@@ -0,0 +1,14 @@
+# Flux functions
+
+Create new functions in Flux code. 
+
+## Required guidelines
+
+Please help us make the contribution process easier by providing feedback about your experience and any technical hurdles you encountered here. 
+
+### **Pure Flux Code Functions Guidelines**
+- Flux functions must conform to examples in [universe.flux](../stdlib/universe/universe.flux).
+- You must submit a unit test in the same folder as the new flux function implementation. 
```

Comment:
I don't think there is any support for unit tests in pure Flux. I think you should remove this line.